docs/org.eclipse.n4js.doc/web-html/articles/index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
```

Comment:
You added a new folder `docs/org.eclipse.n4js.doc/web-html` with many new files. I guess this was by accident.
If this is the case, please remove!

(I noticed that there is a gitignore entry for `web-html`, but only in subfolder `www` of plugin `org.eclipse.n4js.doc`. Maybe there's a glitch in the script for generating the web site?)